### DISCUSSION TITLE
Welcome to emogrifier Discussions!

### DISCUSSION THREAD

**Discussion by jjriv** [jjriv/emogrifier#1178] [Announcements]
Got an email about Hacktoberfest from Digital Ocean and wanted to pass along the info in case you maintainers wanted to participate:
https://hacktoberfest.com/

You might even get a free shirt out of it ;) 

I'm not really a maintainer of this project anymore, so it doesn't matter to me either way. 

Cheers! And thank you for all the work you have been putting into this project.

**Comment by JakeQZ** [comment]
Thanks John.  Good to hear from you.

> I'm not really a maintainer of this project anymore

Figured so.  This is why we need one or two others onboard to keep it going.

[It is used by at least 6% of the top million websites](https://barn2.com/woocommerce-stats/).  So I think we have a certain responsibility to maintain it.  And email clients don't seem to be getting any better, so it's still a valuable peice of software that website owners rely on.

**Reply by JakeQZ** [reply]
> [It is used by at least 6% of the top million websites](https://barn2.com/woocommerce-stats/).

I've not had time to fix https://github.com/woocommerce/woocommerce/issues/30551, since it requires Linux to run the unit tests (though WSL2 may be a way forward).  But it should be fixed.

**Reply by jjriv** [reply]
I should clarify. The emogrifier project matters to me, a lot. What doesn't matter to me is whether or not we promote it on Hacktoberfest. I don't want to presume to tell any of you how to run the project. 

I agree having more maintainers would be great! If there is anything I can do to help recruit, please let me know. 

Meanwhile, thank you for the stats on Woocommerce and emogrifer. That is news to me, and inspiring!

**Reply by oliverklee** [reply]
I have just registered our project to Hacktoberfest (by adding the corresponding project label and issue label) and registered myself as a maintainer and contributor at https://hacktoberfest.com/. 

@JakeQZ Would you be interested in registering at Hacktoberfest as a project maintainer?
